package.json

```diff
@@ -1,25 +1,31 @@
 {
-  "name": "jump-backend-service",
-  "version": "1.0.0",
-  "description": "",
-  "main": "dist/index.js",
-  "types": "dist/index.d.ts",
-  "scripts": {
-    "start": "tsc && node dist/index.js",
-    "build": "tsc",
-    "test": "echo \"Error: no test specified\" && exit 1"
-  },
-  "author": "",
-  "license": "ISC",
-  "dependencies": {
-    "@types/node": "^12.7.2",
-    "typescript": "^3.5.3"
-  },
-  "devDependencies": {
-    "@typescript-eslint/eslint-plugin": "^2.0.0",
-    "@typescript-eslint/parser": "^2.0.0",
-    "eslint": "^6.2.2",
-    "eslint-config-prettier": "^6.1.0",
-    "eslint-plugin-prettier": "^3.1.0"
-  }
+    "name": "jump-backend-service",
+    "version": "1.0.0",
+    "description": "",
+    "main": "dist/index.js",
+    "types": "dist/index.d.ts",
+    "scripts": {
+        "start": "tsc && node dist/index.js",
+        "build": "tsc",
+        "test": "echo \"Error: no test specified\" && exit 1"
+    },
+    "author": "",
+    "license": "ISC",
+    "dependencies": {
+        "@types/node": "^12.7.2",
```

Comment:
If there's a transpile step before running, I think alot of these can be devdependencies.

Reply:
What's a transpile step?

Reply:
Where you compile typescript down to vanilla js. And so the type annotations in typescript won't matter after transpiling because js doesn't have types.